docs/user/boot_options/index.md

```diff
@@ -141,6 +141,13 @@ inst.dud=label://UPDATES/package.rpm
   proxy=http://192.168.122.1:3128
   ```
 
+- `systemd.unit`: standard way to change target for systemd boot. It is useful for head-less mode
+  when wayland neither browsers needs to be started locally.
```

Comment:
It'd rather use more generic term "graphical interface" instead of Wayland.

- It applies to both Wayland and X11.
- Many people are not familiar with the Wayland name yet.